### PR TITLE
Check og:title before title tag

### PR DESCRIPTION
**Pull request by xPaw**
Should play a lot more nicer with `og:description`.

### CODE REVIEW

File: src/plugins/irc-events/link.js [hunk 1]
```diff
@@ -49,7 +49,10 @@ function parse(msg, url, res, client) {
 	case "text/html":
 		var $ = cheerio.load(res.text);
 		toggle.type = "link";
-		toggle.head = $("title").text();
+		toggle.head =
+			$("meta[property=\"og:title\"]").attr("content")
+			|| $("title").text()
+			|| "No title found.";
```
Comment (astorije) [inline src/plugins/irc-events/link.js]: I would actually reword this in `Untitled page` rather than `No title found.` The latter displays like an error message while the former presents a fact about the page. Nitpick, I know.

And because I know this is meant to be consistent with `No description found.`, I would simply remove it. It looks sloppy. If there is no description, let's not show any, it's leaner and doesn't make things obfuscated either.

Comment (xPaw) [inline src/plugins/irc-events/link.js]: Let's instead refuse to show info if there is no preview image/title/description? Or only show "Untitled page" if there is an image. This will need to done in a separate PR though.

Comment (astorije) [inline src/plugins/irc-events/link.js]: Very reasonable.

**Comment by astorije** [comment]
Unfortunately, there are cases where this is not a clear win. For example, when entering https://github.com/rails/rails/search?utf8=%E2%9C%93&q=def+sanitize&type=, one gets:

Before | After
--- | ---
screen shot 2017-07-03 at 01 45 14 | screen shot 2017-07-03 at 01 45 27

The first one comes with more details.
... Also, what's up with that cut width?!

**Comment by astorije** [comment]
Oh my, this width is painful:

On a recent release:

screen shot 2017-07-03 at 01 56 25

On `master`:

screen shot 2017-07-03 at 01 56 41

Completely off-topic of this PR obviously, but have you looked into this by any chance, @xPaw?
